conf/eslint.json

```diff
@@ -19,6 +19,7 @@
         "no-cond-assign": 2,
         "no-console": 2,
         "no-constant-condition": 2,
+        "no-constant-reassign": 0,
```

Comment:
Still think this should be `no-const-reassign` as it relates only to `const` (and also, distinguishes it from `no-constant-condition`, which has nothing to do with `const`).